packages/override-tools/package.json

```diff
@@ -43,6 +42,7 @@
     "@types/prompts": "^2.0.3",
     "@types/yargs": "^15.0.3",
     "babel-jest": "^24.9.0",
+    "diff-match-patch": "^1.0.4",
```

Comment:
I originally thought I was going to use this for main tooling, but this ended up just being used in a script for manifest generation apart from the CLI. Moved it to be a dev dependency.